### DISCUSSION TITLE
How do I spawn new models to increase concurrency? What config file should I edit?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [exadel-inc/CompreFace#866] [Q&A] [answered]
How do I spawn new models to increase concurrency? What config file should I edit?
I have already asked this question, but can't find this discussion.

I need to serve several SDK requests at once. How do I increase amount of simultaneously working models/rest api?

**Comment by pospielov** [comment, marked as answer]
You can tweak it with UWSGI_PROCESSES variable in `compreface-core` container

**Reply by pospielov** [reply]
From the 1.1 version, there will be `uwsgi_processes` setting in `.env` file

**Reply by martinenkoEduard** [reply]
Thanks! It is a good change!
